scripts/stage-release.sh

```diff
@@ -92,10 +92,6 @@ cleanup() {
       gh repo set-default ${ORIGINAL_GH_DEFAULT_REPO}
     fi
 
-    if [[ ${RELEASE_TAG} ]]; then
-      git tag --delete "${RELEASE_TAG}" || true
-    fi
-
```

Comment:
I wasn't sure about deleting this bit versus making `stage-docs.sh` checkout the tag from upstream. The issue with doing the latter is that in dry-run the tag doesn't actually get published to upstream, so that would fail. We could work around it by just skipping the checkout in dry-run, but then the contents of the docs wouldn't necessarily be correct.

Reply:
Removing the code is the right.    The intent of the function is to rollback the changes made locally in dry-run mode.    The tag isn't created in dry-run mode anyway.  Deleting the tag on the dry-run false is just plain wrong.